### DISCUSSION TITLE
Termux app is infected

### DISCUSSION THREAD

**Discussion by wLRy** [termux/termux-app#2950] [General]
Huawei EMUI 10 reports the apk files as infected during install. I checked virustotal and the hashes are also flagged by one of the vendors as infected with "Android:Evo-gen [Trj]":

https://www.virustotal.com/gui/file/3141295849db07a1f4c919ee67a261df723a8ff751b990b87d663a6457b81a2c
https://www.virustotal.com/gui/file/10b412fa0aca5a4585c367a446f8e63525bdc9a7b8dfce2d3f84c2f148e19e21

Is this a false alarm?

**Comment by agnostic-apollo** [comment]
We add a little Code X to our architecture specific builds or is just a false positive.

Duplicate of https://github.com/termux/termux-app/issues/2938
